<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSpeed</class>
 <widget class="QWidget" name="PageSpeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>95</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="layout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelResponseTime">
     <property name="buddy">
      <cstring>sliderResponseTime</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="sliderResponseTime">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelReportRate">
     <property name="buddy">
      <cstring>sliderReportRate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderReportRate">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkGameMode">
     <property name="text">
      <string>&amp;Game mode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
 <connection>
  <sender>sliderReportRate</sender>
  <signal>valueChanged(int)</signal>
  <receiver>PageSpeed</receiver>
  <slot>onReportRateChanged(int)</slot>
  <hints>
   <hint type="sourcelabel">
    <x>20</x>
    <y>20</y>
   </hint>
   <hint type="destinationlabel">
    <x>20</x>
    <y>20</y>
   </hint>
  </hints>
 </connection>
 <connection>
  <sender>sliderResponseTime</sender>
  <signal>valueChanged(int)</signal>
  <receiver>PageSpeed</receiver>
  <slot>onResponseTimeChanged(int)</slot>
  <hints>
   <hint type="sourcelabel">
    <x>20</x>
    <y>20</y>
   </hint>
   <hint type="destinationlabel">
    <x>20</x>
    <y>20</y>
   </hint>
  </hints>
 </connection>
 </connections>
</ui>
